hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson

```diff
@@ -321,6 +321,17 @@
       name: smoke
       tests: ["flash_ctrl_smoke"]
     }
+    {
+      name: jdon
```

Comment:
perhaps don't use username as a regression name

Reply:
whoops, that shouldn't be there.
it was temporary sanitizing regression for this task.
removed it.